python-threatexchange/threatexchange/signal_type/pdq_index.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Implementation of SignalTypeIndex abstraction for PDQ by wrapping hashing.pdq_faiss_matcher.
+"""
+
+import typing as t
+import pickle
+
+from threatexchange.signal_type.index import SignalTypeIndex, IndexMatch
+from threatexchange.hashing.pdq_faiss_matcher import PDQMultiHashIndex
+
+T = t.TypeVar("T")
```

Comment:
If you want to be silly, this also works:

from threatexchange.signal_type import index
T = index.T

Reply:
thoughts on `... import, T as IndexT` ?